### DISCUSSION TITLE
Replaio Radio App

### DISCUSSION THREAD

**Discussion by Mesiyas** [BimmerGestalt/AAIdrive#614] [General]
Hi Hufman, thank you so much for this great app. It's so useful and it works so flawlessly. Absolutely awesome. Thank you again. I wanted to ask you if it would be possible to add Replaio Radio App too?
I mean just like My Tuner, for example. That would be awesome. Thanks again for your amazing work

**Comment by hufman** [comment]
No, unfortunately Replaio enforces a list of which apps are allowed to connect and control it. Please reach out to the Replaio developer to ask them to allow any apps to connect to its MediaBrowserService.

**Reply by Mesiyas** [reply]
Thank you for your fast reply. I have send your answer to the developer. I am hoping for a positive answer

**Comment by Mesiyas** [comment]
@hufman 
Hi Hufman, Replaio has updated the Beta version of the app and now it's work flawlessly with favourites. It's work now like mytuner. 
Thank you so much again for your great work

**Reply by hufman** [reply]
I'm so happy they did that! I wish more music apps would let AAIdrive connect :)

**Reply by Mesiyas** [reply]
I agree with you. And I have to say that Replaio has great support. I forwarded the bug report from you to them and the next day an email came from Replaio that they are working on it. The update came out a few days later

**Comment by hufman** [comment]
I just installed the (non-beta) version of the app and confirm it works, so I added it to the Readme list of supported apps. Thanks for your help!
